signer/src/message.rs

```diff
@@ -322,6 +335,47 @@ pub struct BitcoinTransactionSignAck {
     pub txid: bitcoin::Txid,
 }
 
+/// The message version of an the [`OutPoint`].
+#[derive(Debug, Copy, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct OutPointMessage {
+    /// The referenced transaction's txid.
+    pub txid: bitcoin::Txid,
+    /// The index of the referenced output in its transaction's vout.
+    pub vout: u32,
+}
+
+impl From<OutPoint> for OutPointMessage {
+    fn from(outpoint: OutPoint) -> Self {
+        OutPointMessage {
+            txid: outpoint.txid,
+            vout: outpoint.vout,
+        }
+    }
+}
+
+impl From<OutPointMessage> for OutPoint {
+    fn from(val: OutPointMessage) -> Self {
+        OutPoint { txid: val.txid, vout: val.vout }
+    }
+}
+
+/// The transaction context needed by the signers to reconstruct the transaction.
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct SbtcRequestsContextMessage {
```

Comment:
I'd rename this to `BitcoinPreSignRequest` or something similar.

Later, the context object in the validation file will separate into two, and the meaning of this will change slightly. The signer be validating the information in this object received from the coordinator with "context" information that the receiving signer generates. It'll be similar to how things are done on the stacks side in `contracts.rs`.